regression/goto-harness/chain.sh

```diff
@@ -24,4 +24,4 @@ if [ -e "${name}-mod.gb" ] ; then
 fi
 
 $goto_harness "${name}.gb" "${name}-mod.gb" --harness-function-name $entry_point ${args} 
-$cbmc --function $entry_point "${name}-mod.gb"
+$cbmc --function $entry_point "${name}-mod.gb" --unwind 20 --unwinding-assertions
```

Comment:
Basically it’s a bit annoying to hardcode cbmc options here. We’ve been considering changing our chain.sh to allow us to set cbmc options per test case, but we decided to not do this as part of this PR for now.

Reply:
@hannes-steffenhagen-diffblue - Might be worth raising this point with @karkhaz as he is currently working on re-writing test.pl in Python and cleaning up a few things at the same time - could be a good time to address chain.sh as well.

Reply:
@chrisr-diffblue We were thinking as part of a later PR to change `chain.sh` into a python script. This would make it much easier for things like organising configuration of different tool invocations. We'll see if we can leverage the work done on that front as well.